.github/workflows/chain-spec-snapshot-build.yml

```diff
@@ -26,6 +26,13 @@ jobs:
           file: Dockerfile-node
           push: false
 
+      # On Linux, LLVM 16 is not compatible with the consensus/domain runtime build thus install
+      # LLVM 15 explicitly.
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@c135b3937686fd69c2651507aabc9925a8f9eee8 # v1.8.3
+        with:
+          version: "15.0"
+
```

Comment:
If LLVM wasn't here it doesn't need to be installed. This job runs in a container, it doesn't matter if LLVM is installed on the host at all. Same for other actions where LLVM wasn't already present (but you do need to install it in `rustdoc` action where we didn't have LLVM before).